frontend/pages/404.tsx

```diff
@@ -16,6 +16,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+export const getServerSideProps = () => {
+  return {
+    redirect: {
+      permanent: false,
+      destination: "/",
+    },
+  };
+};
```

Comment:
Was initially thinking of redirecting to 404 page which is why this is implemented here. Maybe I should move it to app.tsx